api-compact.yaml

```diff
@@ -199,3 +274,45 @@ endpoints:
           x-amazon-apigateway-integration:
             <<: *apigateway_integration_with_id_and_querystring_parameters
             uri: "https://${stageVariables.enterprise_host}/enterprise/api/v1/catalogs/{id}/courses/"
+
+    # /v1/enterprise_catalogs/
+    newEnterpriseCatalogs:
```

Comment:
@UmanShahzad Do you know what the existing `enterpriseCatalogs` endpoint is for?